### ISSUE TITLE
Team : Warning dialog before last user leaves

### ISSUE THREAD

**Issue by singharpita** [open-learning-exchange/planet#1708]
isn't the last user should be notified using dialog box after clicking Leave button that Team will be closed or archived???

**Comment by paulbert** [comment]
Yes, this is a good issue.  I'm updating the title a little, but I had the thought that we could implement a dialog that pops up to tell the user to confirm that they will leave & delete this team.

**Comment by RheuX** [comment]
![image](https://github.com/user-attachments/assets/f7088c1c-0f3e-4c78-a48b-a6b0fc03b07b)

should be closed

**Comment by paulbert** [comment]
Fixed in a different way in #4229
